src/Pension/BeamtenPensionRechner.php

```diff
@@ -38,20 +22,19 @@ final class BeamtenPensionRechner
     private const MIN_PENSIONSSATZ_MINDESTRUHEGEHALT    = 0.65;
 
     /**
-     * Enthält das Jahr in dem der Kunde seinen Beamtendienst antritt
-     * @var int|null
+     * Enthält das Jahr, in dem der Kunde seinen Beamtendienst antritt
      */
-    private $dienstzeitbeginn;
+    private ?int $dienstzeitbeginn = null;
 
     /**
      * Enthält das Jahr, in dem der Kunde seinen Beamtendienst beendet und in die
      * Pension eintritt
-     * @var int
      */
-    private $pensionseintritt;
+    private ?int $pensionseintritt = null;
 
     /**
      * @param int $dienstzeitbeginn
+     * @throws Throwable
```

Comment:
Throwable definiert lediglich ein Interface, d.h. die Schnittstelle (a.k.a. welche Methoden oder auch die Form), die eine Exception haben muss. Throwable ist sehr generisch, weil alle möglichen spezifischeren Exceptions meistens dieses Interface implementieren. Eine Exception hilft einem im Idealfall schnell den Fehler herauszufinden, weil sie oftmals bei einem erwarteten Fehler geschmissen wird. In diesem Fall hier sagt ensure zwar das ein Throwable geschmissen wird, aber schaut man sich den Code an, wird eine EnsuranceException geschmissen (siehe Tests). 
Hat PHPStorm vorgeschlagen, dass Throwable hier hinzugefügt wird?

Reply:
> Hat PHPStorm vorgeschlagen, dass Throwable hier hinzugefügt wird?

Genau 👍, war um die weak warnings rauszunehmen

Reply:
@phloose Grundsätzlich gebe ich dir absolut recht, allerdings ist der Übeltäter ja Dgame\Ensurance\ensure die dann mal raus operiert werden sollte. Ich denke aber das sprengt den Rahmen des Tickets :) Ich habe dafür ein Ticket erstellt [LIEBE-205]